pkg/inference/preset-inferences.go

```diff
@@ -63,43 +63,35 @@ var (
 	}
 )
 
-func setTorchParams(ctx context.Context, kubeClient client.Client, wObj *kaitov1alpha1.Workspace, inferenceObj PresetInferenceParam) error {
-	if inferenceObj.ModelName == "LLaMa2" {
-		existingService := &corev1.Service{}
-		err := resources.GetResource(ctx, wObj.Name, wObj.Namespace, kubeClient, existingService)
-		if err != nil {
-			return err
-		}
+func updateTorchParamsForDistributedInference(ctx context.Context, kubeClient client.Client, wObj *kaitov1alpha1.Workspace, inferenceObj *PresetInferenceParam) error {
+	existingService := &corev1.Service{}
+	err := resources.GetResource(ctx, wObj.Name, wObj.Namespace, kubeClient, existingService)
+	if err != nil {
+		return err
+	}
 
-		nodes := *wObj.Resource.Count
-		inferenceObj.TorchRunParams["nnodes"] = strconv.Itoa(nodes)
-		inferenceObj.TorchRunParams["nproc_per_node"] = strconv.Itoa(inferenceObj.WorldSize / nodes)
-		if nodes > 1 {
-			inferenceObj.TorchRunParams["node_rank"] = "$(echo $HOSTNAME | grep -o '[^-]*$')"
-			inferenceObj.TorchRunParams["master_addr"] = existingService.Spec.ClusterIP
-			inferenceObj.TorchRunParams["master_port"] = "29500"
-		}
-		if inferenceObj.TorchRunRdzvParams != nil {
-			inferenceObj.TorchRunRdzvParams["max_restarts"] = "3"
-			inferenceObj.TorchRunRdzvParams["rdzv_id"] = "job"
-			inferenceObj.TorchRunRdzvParams["rdzv_backend"] = "c10d"
-			inferenceObj.TorchRunRdzvParams["rdzv_endpoint"] =
-				fmt.Sprintf("%s-0.%s-headless.%s.svc.cluster.local:29500", wObj.Name, wObj.Name, wObj.Namespace)
-		}
-	} else if inferenceObj.ModelName == "Falcon" {
-		inferenceObj.TorchRunParams["config_file"] = "config.yaml"
-		inferenceObj.TorchRunParams["num_processes"] = "1"
-		inferenceObj.TorchRunParams["num_machines"] = "1"
-		inferenceObj.TorchRunParams["machine_rank"] = "0"
-		inferenceObj.TorchRunParams["gpu_ids"] = "all"
```

Comment:
oh wait where did all this falcon torchrun params go?

Reply:
They are equal to defaults.